packages/opencensus-instrumentation-redis/test/test-redis.ts

```diff
@@ -55,14 +55,30 @@ function assertSpan(
   }
 }
 
+/**
+ * Access the redis client.
+ * @param url The redis URL to access.
+ */
+function accessCollection(url: string): Promise<redis.RedisClient> {
+  return new Promise((resolve, reject) => {
+    const client = redis.createClient({url});
+    client.on('error', (err) => {
+      reject(err);
+      return;
+    });
+    resolve(client);
```

Comment:
you should listen for the `ready` event to be able to resolve the promise i think, otherwise it might resolve once before catching the error.

Reply:
Done, PTAL

Reply:
Fine for me